lib/src/metta/runner/mod.rs

```diff
@@ -948,7 +953,7 @@ impl<'input> RunContext<'_, 'input> {
                 self.load_module_alias(mod_name, mod_id)
             },
             None => {
-                let new_id = self.init_module(mod_name, loader)?;
+                let new_id = self.init_module(mod_name, loader)?;             
```

Comment:
Please don't change whitespaces without purpose:
```suggestion
                let new_id = self.init_module(mod_name, loader)?;
```